src/styles/constants.js

```diff
@@ -4,8 +4,8 @@ export const CONTROL_SIZE = 44;
 export const NAVBAR_SIZE = 58;
 
 /* This color is also used directly in several places in our WebView's CSS. */
-export const BRAND_COLOR = 'hsl(170, 48%, 54%)';
+export const BRAND_COLOR = 'hsl(222, 97%, 69%)';
 export const BORDER_COLOR = BRAND_COLOR;
-export const HIGHLIGHT_COLOR = 'hsla(187, 35%, 51%, 0.5)';
+export const HIGHLIGHT_COLOR = 'hsla(222, 97%, 69%, 0.5)';
```

Comment:
Could you please remind me why we're changing `HIGHLIGHT_COLOR`? In particular, I notice the following:

- Before this commit, `HIGHLIGHT_COLOR` has different HSL values than `BRAND_COLOR`.
- After this commit, `HIGHLIGHT_COLOR` has the same HSL values as `BRAND_COLOR`.

What's going on there? 🙂 If you dig through the [history](https://github.com/zulip/zulip-mobile/blob/master/docs/howto/git.md#reading-history) of the file, can you find any clues about why they might have been different, before?

Reply:
Sorry @chrisbobbe, I didn't notice the fact that they were different previously, will browse through and tell you if there is any reason behind it!

Reply:
After digging the history, I've come to know that this was just an arbitrary value used while reworking on the common styling by Boris in 2017
https://github.com/zulip/zulip-mobile/commit/08c0cbd287904969a2361baa8c8c959e247b80ba#diff-395692415cbb03acdc7f24b1058a24796b7f881c6918ad7a9679cd61f4954863R11

It was then shifted from styles/index.js to the current styles/constants.js file
https://github.com/zulip/zulip-mobile/commit/a4c2c1a6586239d6cc3f2987ba9b29fac57a7570#diff-a81e34235c2a8da54ccf35b5c75a38c728b0c4915f5497604578220b120f9915R8

And during another commit, it was decided to use hsl values instead of rgb ones...
https://github.com/zulip/zulip-mobile/commit/72c0868704aaf383e54cab3de33416bad04f597b#diff-a81e34235c2a8da54ccf35b5c75a38c728b0c4915f5497604578220b120f9915R8

...which introduced decimal values rounded later in this commit giving us the present values for HIGHLIGHT_COLOR (and BRAND_COLOR as well)
https://github.com/zulip/zulip-mobile/commit/4d075a560ca00df95977c5e7d18583857eb5d3f1#diff-a81e34235c2a8da54ccf35b5c75a38c728b0c4915f5497604578220b120f9915R8


So, what should we do now? @chrisbobbe 
Can we stick to using the BRAND_COLOR by decreasing its alpha value? Or should we take a random value again?

Reply:
Great, thanks for finding all that history! I agree that there doesn't seem to be an explanation. Ah, well.

Looking at where HIGHLIGHT_COLOR is used (just in src/common/Touchable.js for when a `Touchable` during the time it's pressed/selected), I'd be very happy with your solution of making it the same as BRAND_COLOR with a different alpha value. It's simpler, and if there's any reason to prefer the changed HSL values based on color theory or good design practices, such a reason is not obvious to me, and it's not documented. 🙂

I think a good, clear way to make this change (such that our work is more helpful to future developers than what we've just found looking in the history so far! 🙂) would be something like this, starting from the beginning of the branch, before the more interesting changes in this PR:

- A commit that changes HIGHLIGHT_COLOR to `hsl(170, 48%, 54%, 0.5)`, i.e., the same as BRAND_COLOR but with the different alpha value. The commit message references and/or summarizes this conversation for why we'd like to make that change.
- Maybe an [`[nfc]` commit](https://github.com/zulip/zulip-mobile/blob/master/docs/glossary.md#nfc-or-nfc) that makes the connection with BRAND_COLOR even more explicit. For example, like this:

  ```js
  import Color from 'color';
  
  // ...
  
  export const HIGHLIGHT_COLOR = Color(BRAND_COLOR)
    .fade(0.5)
    .toString();
  ```
  
  (See the docs of the NPM package `color` that we use: https://www.npmjs.com/package/color#manipulation.)
- Then, the PR proceeds with the color-change from green to blue—except this time HIGHLIGHT_COLOR doesn't have to change; it'll already be computed to be the value we want. 🙂

Reply:
Nice! I will make the necessary changes!